src_cpp/opennnBridge/nerlWorkerOpenNN.cpp

```diff
@@ -569,6 +598,11 @@ namespace nerlnet
             }  
             curr_layer = curr_layer->get_next_layer_ptr();
         }
+        // Write the model parameters to file
```

Comment:
add a dedicated function that saves model parameters.
We should add this capability to NIF and API-Server at some point. 

But remove this comment from here anyway.